.github/workflows/ci.yml

```diff
@@ -98,33 +98,23 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [infinyon-ubuntu-bionic]
         rust: [stable]
-
     steps:
       - uses: actions/checkout@v2
-      - name: Install Helm
-        run: dev-tools/ci-replace-helm.sh
-        env:
-          HELM_VERSION: v3.3.4
-          OS: ${{ matrix.os }}
       - run: helm version
       - name: Install ${{ matrix.rust }}
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.rust }}
           override: true
-      - name: Install Minikube on Linux
-        if: startsWith(matrix.os,'ubuntu')
-        run: |
-          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-          sudo install minikube-linux-amd64 /usr/local/bin/minikube
       - name: Install Minikube on Mac
         if: startsWith(matrix.os, 'macOS')
         run:  brew install minikube
       - name: Setup Minikube for Linux
         if: startsWith(matrix.os, 'ubuntu')
         run: |  
+          minikube delete
```

Comment:
Are there teardowns on github CI? It seems like this would be a good thing to have as a cleanup step as well.

Reply:
well, new CI runs get new VM....

Reply:
Good idea